tar/bsdtar.h

```diff
@@ -175,6 +175,7 @@ enum {
 	OPTION_VERSION,
 	OPTION_XATTRS,
 	OPTION_ZSTD,
+	OPTION_ZZZ_LAST,	/* Added for a sanity check; must be last. */
```

Comment:
Not counting `OPTION_ZZZ_LAST`, there's 61 items in this enum, so there's only space for 1 more until we need a new approach in `bsdtar_getopt()`.

To add a few more spaces, we could add OPTION_UNRECOGNIZED (defined to something like 127) instead of '?', but that only gives a few more potential options -- the next boundary would be 65 'A'.

Reply:
I suppose the other alternative is to renumber these options to begin at say 1000.  That would eliminate any chance of conflict with the ASCII values for single-letter options.

Reply:
Yes; tarsnap (which uses a modified `bsdtar.h` & `bsdtar.c`) starts their "fake short equivalents" enum at 256.

A consequence of that change is that
```
char          compression, compression2;
```
would need to be `int`.

I'd be tempted to change `bsdtar->mode` to an `int` too.  At first glance I don't think it's strictly necessary -- there program modes are only lower-case letters -- but it would reduce any unpleasant surprise in the future if somebody added a long form for a mode.

I'll whip up those changes so you can look at the code and see if you like it.

Reply:
That looks much better.